src/main/java/de/rabea/ContentStorage.java

```diff
@@ -0,0 +1,24 @@
+package de.rabea;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ContentStorage {
+    private Map<Controller, String> storage = new HashMap<>();
+
+    public void store(Controller controller, String body) {
```

Comment:
My thinking was that I need to differentiate which content belongs to which request. 
So if I have a post request to `/route-a`, the content will be saved with the `RouteAController`. 
If I then have another post request to `route-b` it will be saved with the `RouteBController`. 
And when I then receive a get request to `/route-a`, I know to return the content that was saved for `RouteAController` and not the other content.